showpass/Showpass-E2E-tests
# TA-75 Ensure that a seat can be picked when the 'Assigned Seating Selection' is enabled

**Description:**
This is a very challenging test, as the test automation tool is unable to interact with the Canvas.
Cypress is unable to pinpoint elements within the canvas.
This script is more a prototype. It’s designed to just click whatever it is in the middle of the screen.
The problem is that, from time to time it clicks between the dots and, as a result, none of the seats gets assigned and then the test fails.

- Log into the portal as 'qa+158@showpass.com'
- Find an event and click ‘Buy Tickets’
- A canvas should show up during checkout showing available seats
- Pick a seat and ensure the seat was added to cart

NOTE:
This test will be DISABLED to avoid false negatives, as it is very unpredictable and fails sporadically.

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-75

**How to test:**
Find and run the corresponding test within the following spec cypress/e2e/events/verify-events.cy.js

**Test execution results:**

<img width="1711" alt="TA-81-test-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/39e66d0b-ed22-4836-ac41-c351811db205">
